shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -34,6 +34,13 @@
 // returns kInvalidFirstRect, iOS will not show the IME candidates view.
 const CGRect kInvalidFirstRect = {{-1, -1}, {9999, 9999}};
 
+// Position FlutterTextInputView outside of the screen (if scribble is disabled).
+// This is to fix a bug where native auto-correction highlight is displayed on
+// top left corner of the screen (See: https://github.com/flutter/flutter/issues/131695)
+// and a bug where the native auto-correction suggestion menu displayed (See:
+// https://github.com/flutter/flutter/issues/130818).
+const CGRect kNonScribbleInputHiderOffScreenRect = {{0, -2000}, {0, 0}};
```

Comment:
If we need to keep the off-screen hack, please use a dynamic value that is definitively off-screen based on iterating all `UIScreen` objects and uninioning their bounds.

Reply:
Updated to use the current UIScreen that it is on (I think current screen is good enough unless im misunderstanding)